src/primitives/transaction.h

```diff
@@ -359,25 +363,39 @@ class CTxOut
     CAmount nValue;
     CScript scriptPubKey;
 
+    /* mem only */
```

Comment:
remove this. A new method isFromBackwardTransfer(int idx) will be introduced in all CTxOut "containers" to test if a specific output is a bwt.